code/drasil-lang/Language/Drasil/Chunk/DefinedQuantity.hs

```diff
@@ -63,6 +63,9 @@ dqd' = DQD
 dqdWr :: (Quantity c, Concept c, MayHaveUnit c) => c -> DefinedQuantityDict
 dqdWr c = DQD (cw c) (symbol c) (c ^. typ) (getUnit c)
 
+dqdWr' :: (Quantity c, Concept c, HasUnit c) => c -> DefinedQuantityDict
+dqdWr' c = DQD (cw c) (symbol c) (c ^. typ) (Just $ findUnit c)
```

Comment:
This should be marked as a temporary constructor that should not be used outside `drasil-lang`.